### PR TITLE
Alert avatar

### PR DESCRIPTION
**Pull request by varnita619**
made alert component

### CODE REVIEW

File: Components/alert/alert.html [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+                <img src="/logo.svg" class="nav-logo" alt="">
+                <!-- <span class=" ui-version">v1.0</span> -->
+
+                <div class="nav--align-start">
+                    <a href="#" class="nav-links">Home</a>
+                    <a href="#" class="nav-links">Documentation</a>
+                </div>
+
+                <div class="nav--align-end">
+                    <i class="fa fa-moon" aria-hidden="true"></i>
```
Comment (stutisk) [inline Components/alert/alert.html]: hey ! you can try adding title attribute to `i`element. else looks good to me

**Review by VedanthB — APPROVED** [review]
Looks good to go 🚀

File: Components/alert/alert.html [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+    <!-- font awesome link -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.0/css/all.min.css"
+        integrity="sha512-PgQMlq+nqFLV4ylk1gwUOgm6CtIIXkKwaIHp/PAIWHzig/lKZSEGKEysh0TCVbHJXCLN7WetD8TFecIky75ZfQ=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+    <!-- font family link -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Bitter:wght@600&display=swap" rel="stylesheet">
```
Comment (Mohasin-Haque) [inline Components/alert/alert.html:21]: Hey !! You made it nicely .. But may be forgot to format it.. Please do it!!

**Review by Mohasin-Haque — COMMENTED** [review]
Everything seems fine to me. but do care of formatting the code. I have mentioned a comment also.. Good work!

**Review by anuj-kumary — CHANGES_REQUESTED** [review]
One suggestion on alert component try to give font-weight property or strong element to alert text i.e. Error, Success,Warn.

**Comment by varnita619** [comment]
> Everything seems fine to me. but do care of formatting the code. I have mentioned a comment also.. Good work!

Sure, Thanks

File: Components/alert/alert.html (hunk 1 above)
Comment (varnita619) [reply Components/alert/alert.html]: Sure I will do that, thanks

**Comment by varnita619** [comment]
> One suggestion on alert component try to give font-weight property or strong element to alert text i.e. Error, Success,Warn.

Thanks for the suggestion, I have done that,

**Review by anuj-kumary — APPROVED** [review]
Looks good !!

**Pull request merged**